rhino-core/src/main/java/io/ryos/rhino/sdk/SimulationJobsScannerImpl.java

```diff
@@ -220,18 +229,21 @@ private SimulationMetadata createBenchmarkJob(final Class clazz) {
         .withCleanUp(findMethodWith(clazz, CleanUp.class).orElse(null))
         .withBefore(findMethodWith(clazz, Before.class).orElse(null))
         .withAfter(findMethodWith(clazz, After.class).orElse(null))
-        .withScenarios(stepMethods)
-        .withSpecs(specMethods)
-        .withRampUp(-1). // Throttling is not scope of 1.0 anymore.
-        build();
+        .withScenarios(scenarioMethods)
+        .withSpecs(dsls)
+        .withTestInstance(testInstance)
```

Comment:
Test instance is an instance of SimulationMetadata on which we run the reflection voodoo.